src/pybind/mgr/nfs/cluster.py

```diff
@@ -55,27 +56,32 @@ def __init__(self, mgr: 'Module') -> None:
     def _call_orch_apply_nfs(
             self,
             cluster_id: str,
-            placement: Optional[str],
+            placement: Optional[str] = None,
```

Comment:
Conceptually, yes, but this turns out to be quite difficult due to all the different ways placement can be done. Most deployments are not going to just provide "count" since that will place on a random host, so in order to implement this we'd have to be able to verify only 1 matching host for placements by explicit hosts (that one specifically would be easy to be fair), host pattern, and host labels. Host pattern and host label would be quite difficult to implement correctly here. I'd personally prefer to just allow any placement for now, and when this gets documented, just mention that you should really only be deploying 1 nfs daemon with this setup.

Reply:
scratch that, I think I understand what you meant a bit better here. I think this is possible, as long as we're willing to modify the placement a bit on our own.

Reply:
added count:1 to the placements when created through nfs cluster create here